Pod/Classes/PINRemoteImageManager.m

```diff
@@ -751,6 +751,25 @@ - (void)downloadImageWithURL:(NSURL *)url
     [self unlock];
 }
 
+-(BOOL) insertImageDataIntoCache:(nonnull NSData*)data withURL:(nonnull NSURL *)url processorKey:(nullable NSString *)processorKey {
+  
+  if (url != nil) {
+    NSString *key = [self cacheKeyForURL:url processorKey:processorKey];
+    
+    PINRemoteImageManagerDownloadOptions options = PINRemoteImageManagerDownloadOptionsSkipDecode & PINRemoteImageManagerDownloadOptionsSkipEarlyCheck;
+    PINRemoteImageMemoryContainer *container = [[PINRemoteImageMemoryContainer alloc] init];
+    container.data = data;
+    
+    return [self materializeAndCacheObject: container cacheInDisk: data additionalCost:0 key:key options:options outImage: nil outAltRep: nil];
+  }
+  
+  return NO;
+}
+
+- (BOOL) insertImageDataIntoCache:(NSData *)data withURL:(NSURL *)url {
```

Comment:
I'm not sure it's worth exposing this convenience method as it's not likely to be called that often?